ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -27,7 +27,7 @@
 import io.confluent.ksql.parser.tree.SetProperty;
 import io.confluent.ksql.parser.tree.Statement;
 import io.confluent.ksql.parser.tree.UnsetProperty;
-import io.confluent.ksql.rest.util.ProcessingLogConfig;
+import io.confluent.ksql.processing.log.ProcessingLogConfig;
```

Comment:
nit: can we switch this around to `logging.processing`, then we can have other logging stuff under `logging.blah` etc.

Reply:
Sounds good. I'll do it in a follow up. I want to move some of the files/code around anyway.